File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    max_w = 0
    max_h = 0
    
    for size in sizes:
        size.sort()
        for i in range(len(size)):
            if max_w < size[0]:
                max_w = size[0]
            if max_h < size[1]:
                max_h = size[1]
        answer = max_w * max_h
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
import java.util.*;

class Solution {
    public int solution(int[][] sizes) {
        int max_height = 0;
        int max_width = 0;
        
        for (int i = 0; i < sizes.length; i++) {
            Arrays.sort(sizes[i]);
            
            max_height = Math.max(max_height, sizes[i][0]);
            max_width = Math.max(max_width, sizes[i][1]);
        }
        
        return max_height * max_width;
    }
}